js/simplewebrtc.js

```diff
@@ -3886,9 +3886,10 @@
 		}
 
 		if (window.navigator.userAgent.match('Chrome')) {
-			var chromever = parseInt(window.navigator.userAgent.match(/Chrome\/(.*) /)[1], 10);
+			var chromematch = window.navigator.userAgent.match(/Chrome\/(.*)\b/);
```

Comment:
I updated the detection code in https://github.com/nextcloud/spreed/pull/1360/commits/879b94948163102e3ceca5ecd2632544cead139c to properly handle all cases now (Iridium was not matching because `Chrome/XX.YY` was at the end of the user agent).